include/game.hpp

```diff
@@ -0,0 +1,116 @@
+#ifndef GAME_HPP_
+#define GAME_HPP_
+
+#include <json_fwd.hpp>
+#include <map>
+#include <memory>
+#include <utility>
+#include <vector>
+#include "card_research.hpp"
+#include "character.hpp"
+#include "map.hpp"
+#include "runebound_fwd.hpp"
+#include "tokens.hpp"
+
+namespace runebound {
+const int DECK_SIZE = 60;
+
+namespace game {
+void to_json(nlohmann::json &json, const Game &game);
+void from_json(const nlohmann::json &json, Game &game);
+
+struct Game {

```

Comment:
In `C++` world people tend to use `class` for complex classes with methods etc and `struct` for POD/simple structs